### PR TITLE
Meep! Meep meep

### PR DESCRIPTION
**Pull request by lawnsea**
Hey @taravancil and @pfrazee, please have a look and lemme know what issues you find.

### CODE REVIEW

File: _posts/2017-06-20-meetup.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Hey friends! AustinJS is back with an exciting dual talk from Austin's own [Tara Vancil](https://twitter.com/taravancil) and [Paul Frazee](https://twitter.com/pfrazee) about their project, [Beaker Browser][Beaker], an experimental web browser with builtin hosting and p3p web protocols. 
+
+<div class="media-object speaker-bio">
+  <a href="https://twitter.com/taravancil">
+    <img alt="Tara Vancil @taravancil on Twitter"
+      src="https://pbs.twimg.com/profile_images/844315037351096321/qLBLZqkl_400x400.jpg" />
+  </a>
+  <div>
+  <a href="https://twitter.com/taravancil"><strong>Tara Vancil</strong></a>
+  helps build [Beaker][], a browser for the peer-to-peer Web. She’s enthusiastic
```
Comment (lawnsea) [inline _posts/2017-06-20-meetup.md]: @aaronj1335 will this work or do I need an actual `<a>` here?

File: _posts/2017-06-20-meetup.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+layout: post
+categories:
+- posts
+- meetups
+when: 2017-06-20T19:30:00-05:00
+---
+
+<img alt="Beaker" src="https://upload.wikimedia.org/wikipedia/en/5/59/Beaker_%28Muppet%29.jpg" />
+
+Hey friends! AustinJS is back with an exciting dual talk from Austin's own [Tara Vancil](https://twitter.com/taravancil) and [Paul Frazee](https://twitter.com/pfrazee) about their project, [Beaker Browser][Beaker], an experimental web browser with builtin hosting and p3p web protocols.
```
Comment (pfrazee) [inline _posts/2017-06-20-meetup.md]: Is "p3p" a typo? We couldn't decide if it was a typo or just the next version of p2p.

File: _posts/2017-06-20-meetup.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+</div>
+
+<div class="media-object speaker-bio">
+  <a href="https://twitter.com/pfrazee">
+    <img alt="Paul Frazee @pfrazee on Twitter"
+      src="https://pbs.twimg.com/profile_images/868195193253535744/yXQt-VYo_400x400.jpg" />
+  </a>
+  <div>
+  <a href="https://twitter.com/pfrazee"><strong>Paul Frazee</strong></a>
+  works on a peer-to-peer browser called [Beaker][]. He lives in Austin TX.
```
Comment (pfrazee) [inline _posts/2017-06-20-meetup.md]: Small change: "Paul Frazee works on Beaker, and is also very enthusiastic about a decentralized Web. He lives in Austin TX."

File: _posts/2017-06-20-meetup.md (hunk 1 above)
Comment (aaronj1335) [inline _posts/2017-06-20-meetup.md]: Won't work. When in doubt, HTML.

File: _posts/2017-06-20-meetup.md (hunk 2 above)
Comment (aaronj1335) [inline _posts/2017-06-20-meetup.md]: NEXT GEN P2P 😂😂😂

**Comment by aaronj1335** [comment]
A bit disappointed in the complete dearth of dat boi jokes but LGTM otherwise.

**Pull request merged**